eng/testing/tests.wasm.targets

```diff
@@ -109,6 +109,8 @@
       <BundleFiles Include="$(RuntimeConfigFilePath)"           TargetDir="publish" />
 
       <BundleFiles Include="$(WasmSharedPath)data\aot-tests\*"  TargetDir="publish" />
+      <!-- FIXME: what would be the correct place to do this? -->
```

Comment:
should we rename `wasm` -> `browser`  (OS) and use `wasm` for the (architecture) ?

Reply:
Yeah, I avoided those renames in this PR. We are following the pattern you suggested in most places now, like `tests.{wasm,browser,wasi}.targets`, and recently `sendtohelix-{wasm,browser,wasi}.proj`.